src/Orchard.Web/Modules/Orchard.Alias/Implementation/Storage/AliasStorage.cs

```diff
@@ -90,7 +90,7 @@ public class AliasStorage : IAliasStorage {
         }
 
         public void Remove(string path) {
-            Remove(x => x.Path == path && x.Source == path);
+            Remove(x => x.Path == path);

```

Comment:
This fixes the issue where you couldn't delete an alias record from the Alias UI anymore.